recipes/medcoupling/swig_big_file_compile_flag.patch

```diff
@@ -0,0 +1,43 @@
+Subject: [PATCH] swig big file compile flag
+---
+Index: src/RENUMBER_Swig/CMakeLists.txt
+IDEA additional info:
+Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
+<+>UTF-8
+===================================================================
+diff --git a/src/RENUMBER_Swig/CMakeLists.txt b/src/RENUMBER_Swig/CMakeLists.txt
+--- a/src/RENUMBER_Swig/CMakeLists.txt	(revision 5cb0b9f7852dbb4d68059c363cc38377ed5b3ddb)
++++ b/src/RENUMBER_Swig/CMakeLists.txt	(date 1703937028960)
+@@ -70,6 +70,7 @@
+
+ IF(WIN32)
+   SET_TARGET_PROPERTIES(_MEDRenumber PROPERTIES DEBUG_OUTPUT_NAME _MEDRenumber_d)
++  TARGET_COMPILE_OPTIONS(_MEDRenumber PRIVATE /bigobj)
+ ENDIF(WIN32)
+ INSTALL(TARGETS ${SWIG_MODULE_MEDRenumber_REAL_NAME} DESTINATION ${MEDCOUPLING_INSTALL_PYTHON})
+
+Index: src/MEDCoupling_Swig/CMakeLists.txt
+IDEA additional info:
+Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
+<+>UTF-8
+===================================================================
+diff --git a/src/MEDCoupling_Swig/CMakeLists.txt b/src/MEDCoupling_Swig/CMakeLists.txt
+--- a/src/MEDCoupling_Swig/CMakeLists.txt	(revision 5cb0b9f7852dbb4d68059c363cc38377ed5b3ddb)
++++ b/src/MEDCoupling_Swig/CMakeLists.txt	(date 1703937272130)
+@@ -123,6 +123,7 @@
+
+ IF(WIN32 OR CYGWIN)
+   SWIG_LINK_LIBRARIES(${MEDCouling_target_name} ${PYTHON_LIBRARIES} ${PLATFORM_LIBS} medcouplingcpp)
++  TARGET_COMPILE_OPTIONS(medcouplingcpp PRIVATE /bigobj)
```

Comment:
why not just add the flag to CMAKE_CXX_FLAGS instead of patching ?

Reply:
@jschueller Yeah, I agree that would be better :) I can't recall why I ended up making a patch for this, but I'll just re-run this locally with adding the flag now to see that there wasn't a reason why I did end up patching.

Reply:
Adding `-D CMAKE_CXX_FLAGS="/bigobj" ^` did the same trick as my patch. So I'm removing the patch. Thanks!